content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,86 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+We are proud to announce Mixxx 2.4, which will be released NEXT WEEK, a new major version of our free open source DJ software. It took about two years since the last release due to the large number of new features and changes under the hood.
+
+The 2.4 release includes the following highlights:
+
+### EngineOS export
+Mixxx 2.4 adds support for exporting crates, playlists and the library to Engine DJ OS devices (Denon and Numark standalone controllers).
+@Video(https://www.youtube.com/watch?v=Qle9Y1KdD2Y)
+This lets users prepare their tracks on a laptop using Mixxx and export them to a USB stick for use on a standalone controller at their gig.
+
+### Saved Loops
+Mixxx 2.4 adds support for Saved Loops
+If you already have tracks with Saved Loops from Serato DJ, Mixxx 2.4 will automatically read them from the track metadata.
+
+<video autoplay loop><source src="/images/news/SavedLoopsFeature.mp4" type='video/mp4'></video>
+
+### Effects
+Mixxx 2.4 introduces Effect Chains, a set of effects with parameters which can be saved and loaded with all its parameters.
+In the effect units, Mixxx 2.4 visualizes these effect parameter by value, unit and knob position.
+Once you've created an Effect Chain, you can also define individual Super Knob mappings, that allows to use it as Quick-Effect, where you take creative control with a single knob.
+
+### Sync
+Mixxx 2.4 adds support for choosing the sync leader by clicking on the new button with the crown symbol next to the Sync button.
+This mode is useful when tracks have variable BPM and sync is enabled, all the decks will follow the changing BPM of the sync leader.
+
+### Track Menu
+The track menu in Mixxx 2.4 is extended by several new features:
+- An action to select loaded track in library
+- An action to analyze/re-analyze with variable/constant BPM on a per-track basis
+- An action for looking up track metadata at Discogs, Soundcoud and LastFM
+- An action to remove track files from disk
+
+### Many other changes
+
+#### Soundstretcher / Key-Shift Algorithms
+Mixxx 2.4 now supports three different key shift algorithms:
+- Soundtouch (Basic audio quality - high performance)
+- Rubberband R2 (High audio quality - higher CPU usage)
+- Rubberband R3 (HIFI audio quality - very high CPU usage)
```

Comment:
```suggestion
 - Soundtouch (Basic audio quality - high performance)
 - Rubberband R2 (High audio quality - higher CPU usage)
 - Rubberband R3 (HIFI audio quality - very high CPU usage)
```
otherwise the list is not rendered correctly.